en/ui-system/components/editor/videoplayer.md

```diff
@@ -129,9 +129,9 @@ export class VideoPlayerCtrl extends Component {
 
 Likewise, you can also register `meta-loaded`, `clicked`, `playing` events, and the parameters of the callback function for these events are consistent with the `ready-to-play` parameters.
 
-Please refer to the [VideoPlayer API](../../../api/en/classes/VideoPlayer.html) documentation for details on VideoPlayer events..
+Please refer to the [VideoPlayer API](../../../api/en/classes/VideoPlayer.html) documentation for details on VideoPlayer events.
 
-**Note:** As VideoPlayer is a special component, it cannot register `touch` or `mouse` events on the node with **VideoPlayer** component.
+> **Note**: As **VideoPlayer** is a special component, it cannot register `touch` or `mouse` events on the node with **VideoPlayer** component.
```

Comment:
1. As -> as
2. Is the bolded symbol \_\_xx\_\_ or \*\*xx\*\* or both? In Chinese, \*\*xx\*\* is used, and I see that many English documentation use \_\_xx\_\_.

Reply:
the markdown standard says ** and __ are acceptable. https://www.markdownguide.org/basic-syntax/#bold

Reply:
So how about we just unify one format?